.github/workflows/schedule.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         run: python -m pip install tox
 
       - name: Run ruff
-        run: tox -e format
+        run: tox -e format -- --fix
```

Comment:
Won't this suppress the format error, since this fix will never be applied to the source code?

Reply:
Yes，i will never be apply to tests and just set the same format like main.yml

Reply:
Please see https://github.com/edtechre/pybroker/commit/a62e437432fdf9f8e5e7d7ae0478c14d01a0e765

- Adds "check" arg in to Tox commands, and removes these from the .yml files
- Uses --diff along with --fix for format

Reply:
Ok, it work!